platformio.ini

```diff
@@ -496,6 +506,23 @@ build_flags     = -DSPECTRUM=1
                   ${dev_m5stick-c-plus.build_flags}
 board_build.partitions = config/partitions_custom_noota.csv
 
+[env:spectrum2]
+extends         = dev_m5stick-c-plus2
+build_flags     = -DSPECTRUM=1
+                  -DSHOW_VU_METER=1
+                  -DM5STICKCPLUS2=1
```

Comment:
This line can be removed. It's in included in the `dev_m5stick-c-plus2` section build flags.